packages/discord.js/src/structures/Guild.js

```diff
@@ -1418,8 +1409,8 @@ class Guild extends AnonymousGuild {
       this.client.voice.adapters.set(this.id, methods);
       return {
         sendPayload: data => {
-          if (this.shard.status !== Status.Ready) return false;
-          this.shard.send(data);
+          if (this.client.status !== Status.Ready) return false;
+          this.client.ws.send(this.shardId, data);
```

Comment:
ditto

Reply:
we can't really change this into an async method without also changing `@discoerdjs/voice` because that's where these get used. So I'd say that's outside of the scope of my PR...